### ISSUE TITLE
PEAR.ControlStructures.MultiLineCondition.Alignment false-positive with multiline groups

### ISSUE THREAD

**Issue by EvgenyOrekhov** [squizlabs/PHP_CodeSniffer#2013]
test.php

```php
<?php

if (
    $a
    && (
        $b
        || $c
    )
) {

}
```

Result:

```text
FILE: /test.php
------------------------------------------------------------------------------------------------------------------------------
FOUND 2 ERRORS AFFECTING 2 LINES
------------------------------------------------------------------------------------------------------------------------------
 6 | ERROR | [x] Multi-line IF statement not indented correctly; expected 4 spaces but found 8
   |       |     (PEAR.ControlStructures.MultiLineCondition.Alignment)
 7 | ERROR | [x] Multi-line IF statement not indented correctly; expected 4 spaces but found 8
   |       |     (PEAR.ControlStructures.MultiLineCondition.Alignment)
```

Expected result: no warnings.

The indentation should increase by one level when the last token on a line is `(`.

PHP_CodeSniffer version: 3.2.3
PHP version: 7.2.3

**Comment by gsherwood** [comment]
> The indentation should increase by one level when the last token on a line is (.

No it shouldn't. The PEAR standard is pretty clear. See the section `Split long if statements onto several lines` in the published standard: http://pear.php.net/manual/en/standards.control.php

If you want something different, I think you'd need to write a custom sniff in this case as there are no included sniffs that will enforce the indent rule that you are after.

**Comment by EvgenyOrekhov** [comment]
@gsherwood Thanks! I'll stick to naming and splitting the condition sets then.
